src/ucp/dt/dt.c

```diff
@@ -31,10 +31,10 @@ UCS_PROFILE_FUNC_VOID(ucp_mem_type_unpack,
                       ucp_worker_h worker, void *buffer, const void *recv_data,
                       size_t recv_length, ucs_memory_type_t mem_type)
 {
-    ucp_ep_h ep         = worker->mem_type_ep[mem_type];
+    ucp_ep_h ep                 = worker->mem_type_ep[mem_type];
+    ucp_mem_dummy_handle_t memh = {};
```

Comment:
why is this needed?
(also in `ucp_mem_type_pack`)

Reply:
in PR desc I call it "static user memh"

Reply:
why need to initialize it to 0?

Reply:
user memh flow expect it to be zeroed

Reply:
can we avoid this extra overhead?